editor/writing-flow/index.js

```diff
@@ -31,14 +31,14 @@ class WritingFlow extends Component {
 	}
 
 	getVisibleTabbables() {
-		const tabbablesSelector = [
-			'*[contenteditable="true"]',
-			'*[tabindex]:not([tabindex="-1"])',
-			'textarea',
-			'input',
-		].join( ', ' );
-		const isVisible = ( elem ) => elem.offsetWidth > 0 || elem.offsetHeight > 0 || elem.getClientRects().length > 0;
-		return [ ...this.container.querySelectorAll( tabbablesSelector ) ].filter( isVisible );
+		return focus.tabbable
+			.find( this.container )
+			.filter( ( node ) => (
+				node.nodeName === 'INPUT' ||
+				node.nodeName === 'TEXTAREA' ||
+				( node.isContentEditable && node.contentEditable !== 'inherit' ) ||
```

Comment:
Could this be simplified to `node.contentEditable === 'true'` ?

Reply:
sure